static/motd.js

```diff
@@ -34,16 +34,13 @@ function handleMotd(motd, motdNode, subLang, adsEnabled, onHide) {
         motdNode.removeClass('hide');
         motdNode.find(".close")
             .on('click', function () {
-                motdNode.addClass('hide');
+                motdNode.addClass('d-none');
```

Comment:
Argh! The tragedy of no namespaces in CSS :man_facepalming: - I bet that was tricky to find...

Reply:
Not at all! It happens to be a well documented change on bootstrap 4: The creation of the `.d(isplay)-*` classes which deprecate the `hide/shown` ones :)